debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java

```diff
@@ -516,6 +516,27 @@ public void jitAddSpilledRegisters(WalkState walkState, VoidPointer stackMap) th
 					}
 					while (! savedGPRs.eq(0));
 				}
+			} else if (J9ConfigFlags.arch_aarch64) {
+				savedGPRs = registerSaveDescription.bitAnd(new UDATA(0xFFFF0000));
+
+				if (! savedGPRs.eq(0))
+				{
+					saveOffset = registerSaveDescription.bitAnd(new UDATA(0xFFFF));
+					saveCursor = walkState.bp.subOffset(saveOffset);
+					U8 i = new U8(1);
+					do
+					{
+						if (savedGPRs.anyBitsIn(1))
+						{
+							walkState.registerEAs[mapCursor] = saveCursor;
+							saveCursor = saveCursor.add(1);
+						}
+						i = i.add(1);
+						++mapCursor;
+						savedGPRs = savedGPRs.rightShift(1);
+					}
+					while (! savedGPRs.eq(0));
+				}
```

Comment:
It doesn't appear that `i` is needed.
Also, please fix formatting:
```
	if (!savedGPRs.eq(0)) {
		saveOffset = registerSaveDescription.bitAnd(0xFFFF);
		saveCursor = walkState.bp.subOffset(saveOffset);
		do {
			if (savedGPRs.anyBitsIn(1)) {
				walkState.registerEAs[mapCursor] = saveCursor;
				saveCursor = saveCursor.add(1);
			}
			++mapCursor;
			savedGPRs = savedGPRs.rightShift(1);
		} while (!savedGPRs.eq(0));
	}
```

Reply:
Removed `i` and fixed formatting.